app/controllers/components/course/controller_component_host.rb

```diff
@@ -32,6 +32,11 @@ def enabled_by_default?
       def key
         name.underscore.tr('/', '_').to_sym
       end
+
+      # Override this to customise the display name of the component.
+      def display_name
+        name # Use the module name as the default display name.
```

Comment:
Move the comment to the documentation block above, document the return type alos.